assets/javascripts/discourse/components/show-user-notes.hbs

```diff
@@ -1,7 +1,7 @@
-<a {{action show}} href class="btn btn-default">
+<a {{on "click" @show}} href class="btn btn-default">
   {{d-icon "pencil-alt"}}
   {{#if showCount}}
-    {{i18n "user_notes.show" count=count}}
+    {{i18n "user_notes.show" count=@count}}
   {{else}}
     {{i18n "user_notes.title"}}
   {{/if}}
```

Comment:
A little beyond the scope of 'adopt modern ember primatives'... but I wonder, could this be easily converted to a DButton? It feels a little strange that we have an `<a` with no href, and the `btn btn-default` classes.